services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1452,6 +1452,13 @@ static void emitPushJobStatusMetrics(
    */
   @Override
   public void sendPushJobDetails(PushJobStatusRecordKey key, PushJobDetails value) {
+    if (getPushJobStatusStoreClusterName() == null || getPushJobStatusStoreClusterName().isEmpty()) {
```

Comment:
we can also use `Strings.isEmpty(getPushJobStatusStoreClusterName())` or `StringUtils.isNullOrEmpty(getPushJobStatusStoreClusterName())`, without adding new dependencies